django_north/management/commands/migrate.py

```diff
@@ -16,6 +17,21 @@
 class Command(BaseCommand):
     help = "Migrate the DB to the target version."
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--noinput', '--no-input', action='store_false', dest='interactive',
+            help='Tells Django to NOT prompt the user for input of any kind.',
+        )
+        parser.add_argument(
+            '--database', action='store', dest='database',
+            default=DEFAULT_DB_ALIAS,
+            help='Nominates a database to synchronize. Defaults to the "default" database.',
+        )
+        parser.add_argument(
+            '--run-syncdb', action='store_true', dest='run_syncdb',
+            help='Creates tables for apps without migrations.',
+        )
+
```

Comment:
Why do you add these options ? They are not used in the following code.
Warning for the `run-syncdb` option: is it really relevant in a north context ?